packages/appstore/src/components/wallets-consent-popup/wallets-consent-form.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import WalletConsentFormIcon from 'Assets/svgs/wallets';
+
+const WalletsConsentForm = ({ is_eu, is_high_risk }) => (
+    <React.Fragment>
+        <WalletConsentFormIcon className='wallet-wrapper--icon' />
+        <div className='wallet-wrapper--text'>
+            <Text size='m' align='center' weight='bold' line_height='l'>
+                <Localize i18n_default_text='Ready to upgrade?' />
+            </Text>
+            <Text align='center' line_height='l'>
+                <Localize
+                    i18n_default_text="This is <0>irreversible.</0> Once you upgrade, the Cashier won't be available anymore. You'll need to
+                use Wallets to deposit, withdraw, and transfer funds."
+                    components={<Text weight='bold' align='center' line_height='l' key={0} />}
+                />
+            </Text>
+        </div>
+        <div className='wallet-wrapper--info-section'>
```

Comment:
please define an array for this information part and map them all here. Sth like below
```suggestion
        const geUpgradetInformation = () => [
        {
        name: 'open_positions,
         is_visible: true, 
         description: <Localize
                                i18n_default_text='During the upgrade, deposits, withdrawals, transfers, and adding new accounts will be
                                  unavailable.'
                            />
          },
          ..., 
          {
               name:''payment_agent',
               is_visible: !is_eu,
               description: <Localize i18n_default_text="You can use payment agents' services to make deposits by adding a Payment agent wallet after the upgrade." />
          }
                         
        ]
```

Reply:
Done 😃 👍🏼